package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "setup-dbs": "psql -f ./db/setup.sql",
-    "seed": "node ./db/seeds/run-seed.js",
-    "test": "jest",
+    "seed": "npm run setup-dbs && node ./db/seeds/run-seed.js",
+    "test": "npm run seed && jest",
```

Comment:
remove the seed command from line 9 - this should be done by your test file via the beforeEach hook